src/types/Options.ts

```diff
@@ -0,0 +1,5 @@
+export enum Options{
```

Comment:
```suggestion
export enum TodosFilterOptions {
```
I would prefer this more specific name